### DISCUSSION TITLE
plugin.http.include_http_headers

### DISCUSSION THREAD

**Discussion by aaHorse** [apache/skywalking#8176] [Q&A]
We set up this configuration **plugin.http.include_http_headers=trace-8-id** but our trace does not collect data.What is javax.servlet.http?
![image](https://user-images.githubusercontent.com/38253852/143049173-411499d1-4f9e-47cb-ad65-43006bcbc1b6.png)

**Comment by wu-sheng** [comment]
> What is javax.servlet.http?

What does this question mean? This is Java JEP level definition. Why ask us?

>  plugin.http.include_http_headers=trace-8-id 

AFAIK, this configuration only works for SpringMVC plugin. And I don't know what `trace-8-id ` is, it is definitely on your own.
